Artesian/_GMEPublicOffers/GMEPOfferQuery.py

```diff
@@ -1,7 +1,6 @@
-
 from types import GeneratorType
-from Artesian import _ClientsExecutor
-from Artesian._ClientsExecutor import RequestExecutor
+from Artesian import _Client
```

Comment:
are you sure about this? I'd had expected an import from Artesian._ClientExecutor.
Check if VSCode doesn't complain